ktor-server/ktor-server-cio/posix/src/io/ktor/server/cio/internal/CoroutineUtilsNix.kt

```diff
@@ -7,4 +7,6 @@ package io.ktor.server.cio.internal
 import kotlinx.coroutines.*
 
 internal actual val Dispatchers.IOBridge: CoroutineDispatcher
-    get() = Default
+    get() = IO
```

Comment:
Hm, not sure that it will give a big difference.
Anyway, I will extract it into a separate PR which will use `IO` dispatcher for native in one or two other places and create an issue for this.